tests/travis/install-mysql-5.7.sh

```diff
@@ -2,21 +2,14 @@
 
 set -ex
 
-echo "Installing MySQL 5.7..."
+echo "Starting MySQL 5.7..."
 
-sudo service mysql stop
-sudo apt-get remove "^mysql.*"
-sudo apt-get autoremove
-sudo apt-get autoclean
-echo mysql-apt-config mysql-apt-config/select-server select mysql-5.7 | sudo debconf-set-selections
-wget http://dev.mysql.com/get/mysql-apt-config_0.8.9-1_all.deb
-sudo DEBIAN_FRONTEND=noninteractive dpkg -i mysql-apt-config_0.8.9-1_all.deb
-sudo rm -rf /var/lib/apt/lists/*
-sudo apt-get clean
-sudo apt-get update -q
-sudo apt-get install -q -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" mysql-server libmysqlclient-dev
-sudo mysql_upgrade
-
-echo "Restart mysql..."
-sudo mysql -e "use mysql; update user set authentication_string=PASSWORD('') where User='root'; update user set plugin='mysql_native_password';FLUSH PRIVILEGES;"
+sudo docker run \
+    -d \
+    -e MYSQL_ALLOW_EMPTY_PASSWORD=yes \
+    -e MYSQL_DATABASE=doctrine_tests \
+    -p 33306:3306 \
+    --name mysql57 \
+    mysql:5.7
```

Comment:
Should a fixed 5.7 version/tag be used here or does the build always have to use the newest 5.7 version? I don't expect problems though.

Reply:
We have the same for all other images except for DB2 (not sure if it was tagged deliberately). Unless stability issues start showing up, I think testing against the latest is fine.